src/components/common/TmSessionImport.vue

```diff
@@ -165,9 +168,19 @@ export default {
     }
   }),
   computed: {
-    ...mapGetters([`connected`])
+    ...mapGetters([`connected`]),
```

Comment:
```suggestion
    ...mapGetters([`connected`, `extension`]),
```

that replaces all of `inExtension` I think

Reply:
This will cause an error when the getter `extension` is not defined in the extension. It will work but an error will be in the console. Think it's worth it?

Reply:
How about defining the getter `extension` in the extension?

Reply:
Good idea, this works well.